src/main.js

```diff
@@ -1 +1,40 @@
+import {createProfileTemplate} from "./view/profile.js";
+import {createSiteMenuTemplate} from "./view/site-menu.js";
+import {createFilmsContainerTemplate} from "./view/films-container.js";
+import {createFilmCardTemplate} from "./view/film-card.js";
+import {createShowMoreButtonTemplate} from "./view/show-more-button.js";
+import {createPopupTemplate} from "./view/popup.js";
 
+function render(container, template, place) {
+  container.insertAdjacentHTML(place, template);
+};
+
+const siteMainElement = document.querySelector(`.main`);
+const headerMainElement = document.querySelector(`.header`);
+const footerMainElement = document.querySelector(`.footer`);
+
+render (headerMainElement, createProfileTemplate(), `beforeend`);
```

Comment:
по идее линтер не должен такие вещи пропускать, но стиль должен быть единым во всем проекте

```suggestion
render(headerMainElement, createProfileTemplate(), `beforeend`);
```